src/scriptdev2/scripts/eastern_kingdoms/zulgurub/boss_jeklik.cpp

```diff
@@ -215,24 +238,51 @@ struct boss_jeklikAI : public ScriptedAI
             if (m_uiSwoopTimer < uiDiff)
             {
                 if (DoCastSpellIfCan(m_creature->getVictim(), SPELL_SWOOP) == CAST_OK)
+                {
+                    if (m_creature->getThreatManager().getThreat(m_creature->getVictim()))
+                        m_creature->getThreatManager().modifyThreatPercent(m_creature->getVictim(), -10);
+
                     m_uiSwoopTimer = urand(4000, 9000);
+                }
             }
             else
                 m_uiSwoopTimer -= uiDiff;
 
             if (m_uiSonicBurstTimer < uiDiff)
             {
                 if (DoCastSpellIfCan(m_creature, SPELL_SONIC_BURST) == CAST_OK)
+                {
```

Comment:
Sadly it is, yeah.... so many unneeded diffs just to make the code style of the functions WORSE.... -.-